app/coffeescript/components/ui/markers.coffee

```diff
@@ -64,6 +62,16 @@ define [
       @updateListingsCount()
       @trigger 'uiSetMarkerInfoWindow'
 
+    @clearAllMarkers = ->
```

Comment:
Not sure I understand what you're suggesting. Would you show me the code you have in mind?